bipen/src/main/java/com/lftechnology/phpjava/assignmenttwo/OptionFactory.java

```diff
@@ -0,0 +1,57 @@
+package com.lftechnology.phpjava.assignmenttwo;
+
+import java.util.Scanner;
+
+/**
+ * Class to initiate object according to the users input
+ * 
+ * @author bipen
+ *
+ */
+public class OptionFactory {
+    protected int choosenInput;
+
+    /**
+     * Method to initialize respective class according to user input
+     * 
+     * @param option
+     */
+    public void switchTask(int option) {
+        switch (option) {
+        case 1:
+            SalutationSplitter salutationSplitter = new SalutationSplitter();
+            salutationSplitter.run();
+            break;
+        case 2:
+            Multiplier multiplier = new Multiplier();
+            multiplier.run();
+            break;
+        case 0:
+            System.out.println("Exiting the program now....DONE");
+            System.exit(0);
+        default:
+            System.err.println("Invalid Choosen Number");
+        }
+
+        this.goToMainMenu();
+    }
+
+    /**
+     * Displays Menu to users with option to exit or go back to main menu.
+     */
+    public void goToMainMenu() {
+        System.out.println("Go to Main Menu?(y/n)");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)